docs/topics/booleans.md

```diff
@@ -2,35 +2,43 @@
 
 The type `Boolean` represents boolean objects that can have two values: `true` and `false`. 
 
-> On the JVM, booleans stored as primitive type: `boolean`, typically use 8 bits.
+> On the JVM, booleans stored as the primitive `boolean` type typically use 8 bits.
 >
 {type="note"}
 
-`Boolean` has a nullable counterpart `Boolean?` that also has the `null` value.
+`Boolean` has a nullable counterpart (`Boolean?`) with the `null` value.
 
 Built-in operations on booleans include:
 
 * `||` – disjunction (logical _OR_)
 * `&&` – conjunction (logical _AND_)
 * `!` – negation (logical _NOT_)
 
-`||` and `&&` work lazily.
+For example:
 
 ```kotlin
 fun main() {
 //sampleStart
     val myTrue: Boolean = true
     val myFalse: Boolean = false
     val boolNull: Boolean? = null
```

Comment:
I understand that this is supposed to demonstrate how a `Boolean?` can be null, but it looks like an unused variable. Even the playground points this out. Maybe printing it out together would be clearer? Show that 1) it compiles; 2) the value is actually null.